examples/default.yaml

```diff
@@ -19,6 +19,9 @@ os: null
 # 🟢 Builtin default: "default" (corresponds to the host architecture)
 arch: null
 
+# Name of image to use.
+image: ""
```

Comment:
An alternative design to this PR would be to support extending YAML

e.g.
```yaml
extend: "template://ubuntu"
```

I think there was a GitHub issue, but I can't find it.

Reply:
Another alternative is copy/paste, but they are both slightly more "technical"

You might also inherit more settings than just images, by including/extending?

Reply:
> You might also inherit more settings than just images, by including/extending?

Yes